src/main/java/net/dries007/tfc/common/blockentities/rotation/WindmillBlockEntity.java

```diff
@@ -36,9 +39,11 @@ public class WindmillBlockEntity extends TickableInventoryBlockEntity<ItemStackH
     public static final int SLOTS = 5;
     public static final float MIN_SPEED = Mth.TWO_PI / (20 * 20);
     public static final float MAX_SPEED = Mth.TWO_PI / (8 * 20);
-
     private static final float LERP_SPEED = MIN_SPEED / (5 * 20);
 
+    // client-only
+    public boolean hasFullIdenticalSet = false;
```

Comment:
```suggestion
    // client-only
    private boolean hasFullIdenticalSet = false;
```
Prefer adding an accessor, can even use the existing method for it.